### ISSUE TITLE
Pod is causing App Store rejection

### ISSUE THREAD

**Issue by bdrelling** [yeahdongcn/RSBarcodes_Swift#88]
At some point, the sample was included in the pod directories, causing our app to get rejected when submitting to the app store with these two messages:

```
Missing Info.plist key - This app attempts to access privacy-sensitive data without a usage description. The app's Info.plist must contain an NSPhotoLibraryUsageDescription key with a string value explaining to the user how the app uses this data.

Missing Info.plist key - This app attempts to access privacy-sensitive data without a usage description. The app's Info.plist must contain an NSCameraUsageDescription key with a string value explaining to the user how the app uses this data.
```

Using this link, I looked up references to the mentioned API Requiring Usage Description: https://developer.apple.com/library/content/qa/qa1937/_index.html

The only instance of `AVCaptureDeviceInput` is in the `RSCodeReaderViewController.swift` within `RSBarcodesSample`.

**Comment by yeahdongcn** [comment]
Yes, you have to add NSPhotoLibraryUsageDescription and NSCameraUsageDescription to your app's info.plist if you integrate RSBarcodes_Swift in your app.

**Comment by bdrelling** [comment]
Any logic trying to do this should just be added as an optional subspec to the pod. This isn't something that should be forced on someone just consuming the pod.
